docs-ref-services/latest/messaging-eventgrid-readme.md

```diff
@@ -0,0 +1,470 @@
+---
+title: Azure Event Grid client library for Java
+keywords: Azure, java, SDK, API, azure-messaging-eventgrid, eventgrid
+author: maggiepint
+ms.author: magpint
+ms.date: 03/12/2021
+ms.topic: article
+ms.prod: azure
+ms.technology: azure
+ms.devlang: java
+ms.service: eventgrid
+---
```

Comment:
I see some warnings in the header section. Are they obsolete false alarm?

Line 2, Column 1: [Warning-ms-prod-and-service] Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.
Line 9, Column 16: [Warning-ms-prod-technology-invalid] Invalid value for 'ms.technology': 'azure' is not valid with 'ms.prod' value 'azure'.
Line 11, Column 13: [Warning-ms-service-subservice-invalid] Invalid value for 'ms.service': 'eventgrid'.
Line 2, Column 1: [Suggestion-description-missing] Missing required attribute: 'description'.
Line 37, Column 3: [Suggestion-preserve-view-not-set] Did you mean to link to a specific version? It's usually better not to specify a version, so the link always goes to the canonical version. If you really need to link to a specific version, add &preserve-view=true to the URL: https://docs.microsoft.com/java/azure/jdk/?view=azure-java-stable. Otherwise, remove the view parameter.